Formula/mesa.rb

```diff
@@ -27,10 +27,11 @@ class Mesa < Formula
   depends_on "libxcb"
   depends_on "libxdamage"
   depends_on "libxext"
+  depends_on "llvm"
```

Comment:
Why is this needed? A dependency change on a patch release seems quite unlikely.

Reply:
It's not a hard dependency, but rather a delayed fix. I see more targets are being built with brew `llvm`, i.e. `mesa` doesn't like stock `llvm` for some reason.

Reply:
And `homebrew/core` discourage `recommended` and `optional` dependencies.

Reply:
That seems to apply only on Linux and not macOS. What kind of issues do we see from not having Homebrew LLVM as a dependency?

Reply:
I tried to collect facts, but the only difference is that `LLVM` is being "found", ≈100 extra steps in `ninja`, but files count in resulting bottle is same, and they have mostly same sizes, so I moved `llvm` to `on_linux` with comment.